TASK1/number.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+  if(isset($_POST['submit'])){
+    // If submit is set display the phone number
+    echo "The number is ". $_POST['phone'];
```

Comment:
Indentation in this file is incorrect.
It looks unnecessary to create a separate file just for this purpose.